docs/apache-airflow-providers-google/operators/cloud/kubernetes_engine.rst

```diff
@@ -131,6 +131,14 @@ And then use it in other operators:
     :start-after: [START howto_operator_gke_xcom_result]
     :end-before: [END howto_operator_gke_xcom_result]
 
+You can use deferrable mode for this action in order to run the operator asynchronously:
```

Comment:
There are some limitations of using async KPO in its current state?

What are they?

Reply:
Could you be more descriptive about the idea you propose? I am not sure what you meant. That string is made for docs reader to inform them that deferrable mode is implemented for that pattern of usage